themes/kairos-theme/README.md

```diff
@@ -0,0 +1,15 @@
+# Kairos Website Theme
+
+The Kairos website theme is a one-page-portfolio based on the self-named template made by [Justice](https://freehtml5.co/demos/justice/). Noteworthy features of this Hugo theme are a custom about section, a team section and a contact form.
+
+## Getting Started
+
+Inside the theme folder run:
```

Comment:
Oi? There's no hugo involved in your getting started?

Reply:
I also don't happen to know what `snap` is so maybe I would recommend adding a line saying something like: "Make sure you have snap installed" and a link to instructions on how to install snap :)

Reply:
:+1: